src/main/java/com/laserfiche/repository/api/RepositoryApiClientImpl.java

```diff
@@ -56,6 +56,8 @@ public static RepositoryApiClient CreateFromAccessKey(String servicePrincipalKey
     @Override
     public void setDefaultRequestHeaders(Map<String, String> defaultRequestHeaders) {
         defaultHeaders = defaultRequestHeaders;
+        String defaultHeadersKey = defaultHeaders.keySet().toArray()[0].toString();
+        httpClient.config().addDefaultHeader(defaultHeadersKey, defaultHeaders.get(defaultHeadersKey));
```

Comment:
If `defaultRequestHeaders` has multiple headers, only the first is added?

Reply:
good catch, fixed